contributing.md

```diff
@@ -64,14 +62,67 @@ contains any additional descriptive information.
 
 1. Thank you for submitting changes to Experimenter :D
 
+## Git Commit Guidelines
+
+### Subject
+
+The subject should follow the this pattern:
+
+`fixes #github_issue_number type(scope): Description`
+
+#### Type
+One of the following
+
+- **feat**: A new feature
+- **fix**: A bug fix
+- **docs**: Documentation only changes
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+- **refactor**: A code change that neither fixes a bug or adds a feature
+- **perf**: A code change that improves performance
+- **test**: Adding missing tests
+- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
+
+#### Scope
+
+One of the following:
+
+- **example scope**
```

Comment:
Open to suggestions here, not sure what the granularity should be like? Like maybe

- project (project wide changes)
- dev (things that impact dev environments?)
- backend
- frontend
- ...others?

Reply:
Oh not sure how I glossed over this section 😂

Project is definitely a good one. Something like `docs(nimbus-ui):` or `feat(nimbus-ui)` is pretty descriptive, though for BE changes I'm not sure what you'd call that, `feat(backend)` would maybe work there?